packages/components/src/tooltip/test/index.tsx

```diff
@@ -105,6 +105,33 @@ describe( 'Tooltip', () => {
 				screen.getByRole( 'button', { name: 'Anchor' } )
 			).not.toHaveAttribute( 'data-foo' );
 		} );
+
+		it( 'should add custom class to the tooltip', async () => {
+			render( <Tooltip { ...props } className='foo' /> );
```

Comment:
Nit: With string literals as component props, we usually use double quotes:

```suggestion
			render( <Tooltip { ...props } className="foo" /> );
```